parametric_stellarator.py

```diff
@@ -205,6 +207,27 @@ def exports(export, components, magnets, logger):
         logger.info('Exporting STEP files...')
         for name, comp in components.items():
             cq.exporters.export(comp['solid'], name + '.step')
+            
+        # Conditionally export tetrahedral meshing
+        if magnets['meshing']:
+            # Get current working directory
+            cwd = os.getcwd()      
+            # Exodus export
+            exo_path = f'{cwd}/coil_mesh.exo'
```

Comment:
Define the `base_name` of the file
```
base_name = 'coil_mesh'
exo_path = f'{file_path}/{base_name}.exo'
```
and use that in both places.  This will facilitate changing it in the future.